tests/integration/bigquery_project/dataform.json

```diff
@@ -2,5 +2,7 @@
   "warehouse": "bigquery",
   "defaultDatabase": "dataform-integration-tests",
   "defaultSchema": "df_integration_test",
-  "assertionSchema": "df_integration_test_assertions"
+  "assertionSchema": "df_integration_test_assertions",
+  "useRunCache": true,
+  "useSingleQueryPerAction": true
```

Comment:
IMO i'd just set this in the tests themselves; that would be consistent with run caching

Reply:
Nice